config/global.json

```diff
@@ -94,7 +94,7 @@
     "gravity": "https://safe.gravity.xyz/txs/api",
     "immutablezkevm": "https://transaction.safe.immutable.com/api",
     "kaia": "https://docs-safe.kaia.io/txs-cypress/api",
-    "linea": "https://transaction.safe.linea.build/api",
+    "linea": "https://safe-transaction-linea.safe.global/api",
```

Comment:
if the safe API url is now part of `networks.json`, why do we also still have it in `global.json`?